### ISSUE TITLE
[Pronunciation] Make `ng` and `wh` single characters for the "default"

### ISSUE THREAD

**Issue by kmahelona** [TeHikuMedia/corpora#234]

**Comment by kmahelona** [comment]
Hey @hlm628 I wonder if we can just "do a trick" where we still include the `n` and the `g` but I "bind" the two together. So in the UI tapping one will tap the other, but also the metadata will be similar to what we're doing on the mobile app where the two letters will have the same confidences.

**Comment by hlm628** [comment]
Yes, I think that's ok. As long as I can still unambiguously see that these are "ng" or "wh" from the target sentence (where the digraphs should be published)
